style.css

```diff
@@ -112,6 +114,27 @@ overflow-y:scroll;
     gap:2em;
 }
 
+#greenbox {
+background-color:#5CDB95;
+width:30px;
+height:30px;
+padding:10px;
+}
+
+#bluebox {
+background-color:#45A29E;
+width:30px;
+height:30px;
+padding:10px;
+}
+
+#blackbox {
+background-color:#1F2833;
+width:30px;
+height:30px;
+padding:10px;
```

Comment:
since these properties are the same for all 3 boxes, you could put them in the `.colourlegend` class selector to avoid repetitive lines